mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -45,14 +45,18 @@
 
 CONTENT_TYPE_CSV = "text/csv"
 CONTENT_TYPE_JSON = "application/json"
-CONTENT_TYPE_JSON_RECORDS_ORIENTED = "application/json; format=pandas-records"
-CONTENT_TYPE_JSON_SPLIT_ORIENTED = "application/json; format=pandas-split"
+CONTENT_TYPE_JSON_RECORDS_ORIENTED = "application/json-pandas-records"
+CONTENT_TYPE_JSON_SPLIT_ORIENTED = "application/json-pandas-split"
+CONTENT_TYPE_JSON_RECORDS_NUMPY = "application/json-numpy-records"
```

Comment:
Don't you need to update the docs for this: https://mlflow.org/docs/latest/models.html#deploy-mlflow-models

Reply:
I'm not sure if I need to, I will check with Databricks team on this.